docs/task-files.md

```diff
@@ -304,6 +305,7 @@ The ``update-serverless.com`` task will deploy the [serverless.com](https://serv
 |Skip| `true` or `false` |When `true` task (and dependent tasks) will not be executed.|
 |TaskRoleName|string|Specifies the name of the IAM Role that must be used for cross account access. A role with this is expected to exist in the target account (and have the right AssumeRole permissions).|
 |Parameters|any|Specifies parameters that must be passed to the serverless deployment using command arguments.|
+|MaxConcurrentStacks|number|The number of stacks that should be executed concurrently.<br/><br/> Default = 1|
```

Comment:
For udpate-serverless and update-cdk i believe the setting is called MaxConcurrentTasks.
Also: the setting, currently, only allows for its default value of 1. not sure whether it makes sense to have these attributes listed

Reply:
> For udpate-serverless and update-cdk i believe the setting is called MaxConcurrentTasks.

Hmm,  the example just below has `MaxConcurrentStacks`?  

> Also: the setting, currently, only allows for its default value of 1. not sure whether it makes sense to have these attributes listed

Do you mean the only value that you can set for this is 1?  Does this same limitation also apply to `update-cdk`?